--- gdk-pixbuf/pixops/have_mmx.S.orig	2010-10-08 14:03:55.277000000 +0200
+++ gdk-pixbuf/pixops/have_mmx.S	2010-10-08 14:04:20.633000000 +0200
@@ -42,7 +42,7 @@ __pixops_have_mmx:
 	
 	push	%ebx
 
-# Check if bit 21 in flags word is writeable
+/* # Check if bit 21 in flags word is writeable */
 
 	pushfl	
 	popl	%eax
@@ -57,7 +57,7 @@ __pixops_have_mmx:
 
 	je .notfound
 
-# OK, we have CPUID
+/* # OK, we have CPUID */
 
 	movl	$1, %eax
 	cpuid
--- gdk-pixbuf/pixops/composite_line_22_4a4_mmx.S.orig	2010-10-08 14:08:07.345000000 +0200
+++ gdk-pixbuf/pixops/composite_line_22_4a4_mmx.S	2010-10-08 14:08:28.478000000 +0200
@@ -188,10 +188,10 @@ __pixops_composite_line_22_4a4_mmx:
 	movq %mm3, %mm2
 
 	shll $2, %edx
-
+/*
 #	%mm4 will always be already clear here	
 #	pxor %mm4, %mm4
-
+*/
 	movl 16(%ebp),%edi
 	movl (%edi, %edx), %eax
 	movd (%edi, %edx), %mm5
