--- gio/ginetaddress.c.orig	2011-11-15 15:29:45.395000000 +0100
+++ gio/ginetaddress.c	2011-11-15 15:30:17.270000000 +0100
@@ -485,7 +485,7 @@ g_inet_address_new_loopback (GSocketFami
       return g_inet_address_new_from_bytes (addr, family);
     }
   else
-    return g_inet_address_new_from_bytes (in6addr_loopback.s6_addr, family);
+    return NULL;
 }
 
 /**
@@ -512,7 +512,7 @@ g_inet_address_new_any (GSocketFamily fa
       return g_inet_address_new_from_bytes (addr, family);
     }
   else
-    return g_inet_address_new_from_bytes (in6addr_any.s6_addr, family);
+    return NULL;
 }
 
 
--- gio/gwin32appinfo.c.orig	2011-11-15 17:40:47.254375000 +0100
+++ gio/gwin32appinfo.c	2011-11-15 16:17:09.988750000 +0100
@@ -33,7 +33,7 @@
 #include "glibintl.h"
 
 #include <windows.h>
-#include <shlwapi.h>
+// #include <shlwapi.h>
 
 
 #ifndef ASSOCF_INIT_BYEXENAME
--- gio/gnetworkingprivate.h.orig	2011-11-15 17:39:19.363750000 +0100
+++ gio/gnetworkingprivate.h	2011-11-15 16:13:56.410625000 +0100
@@ -35,7 +35,7 @@
  * inline workarounds for getaddrinfo, getnameinfo and freeaddrinfo if
  * they aren't present at run-time (on Windows 2000).
  */
-#include <wspiapi.h>
+// #include <wspiapi.h>
 #endif
 
 #else /* !G_OS_WIN32 */
--- gio/gthreadedresolver.c.orig	2012-04-20 14:21:24.406375000 +0200
+++ gio/gthreadedresolver.c	2012-04-20 14:21:37.656375000 +0200
@@ -34,6 +34,9 @@
 #include "gsimpleasyncresult.h"
 #include "gsocketaddress.h"
 
+#include <winsock2.h>
+#include <ws2tcpip.h>
+
 G_DEFINE_TYPE (GThreadedResolver, g_threaded_resolver, G_TYPE_RESOLVER)
 
 static void threaded_resolver_thread (gpointer thread_data, gpointer pool_data);
