*** configure.orig	2011-08-23 10:28:30.000015000 -0400
--- configure	2012-03-09 14:02:57.000015000 -0500
***************
*** 2613,2618 ****
--- 2613,2624 ----
  IFS=$ac_save_IFS
  case $host_os in *\ *) host_os=`echo "$host_os" | sed 's/ /-/g'`;; esac
  
+ case $host in
+   *solaris*)
+     mkdir -p $host
+     cp install-sh $host
+     ;;
+ esac
  
  { $as_echo "$as_me:${as_lineno-$LINENO}: checking target system type" >&5
  $as_echo_n "checking target system type... " >&6; }
***************
*** 16338,16344 ****
  	@ HOST="\$(HOST)\" \\\\\\
  	; test ".\$\$HOST" = "." && HOST=$x sh $ax_enable_builddir_auxdir/config.guess $x \\\\\\
  	; BUILD=$x grep "^#### \$\$HOST " Makefile | sed -e 's/.*|//' $x \\\\\\
! 	; use=$x basename "\$\@" -all $x; n=$x echo \$\$BUILD | wc -w $x \\\\\\
  	; echo "MAKE \$\$HOST : \$\$n * \$\@"; if test "\$\$n" = "0" ; then : \\\\\\
  	; BUILD=$x grep "^####.*|" Makefile |tail -1| sed -e 's/.*|//' $x ; fi \\\\\\
  	; test ".\$\$BUILD" = "." && BUILD="." \\\\\\
--- 16344,16350 ----
  	@ HOST="\$(HOST)\" \\\\\\
  	; test ".\$\$HOST" = "." && HOST=$x sh $ax_enable_builddir_auxdir/config.guess $x \\\\\\
  	; BUILD=$x grep "^#### \$\$HOST " Makefile | sed -e 's/.*|//' $x \\\\\\
! 	; use=$x basename "\$\@" -all $x; n=$x echo \$\$BUILD | wc -w | tr -d ' ' $x \\\\\\
  	; echo "MAKE \$\$HOST : \$\$n * \$\@"; if test "\$\$n" = "0" ; then : \\\\\\
  	; BUILD=$x grep "^####.*|" Makefile |tail -1| sed -e 's/.*|//' $x ; fi \\\\\\
  	; test ".\$\$BUILD" = "." && BUILD="." \\\\\\
***************
*** 16371,16377 ****
  	@ HOST="\$(HOST)\" \\\\\\
  	; test ".\$\$HOST" = "." && HOST=$x sh $ax_enable_builddir_auxdir/config.guess $x \\\\\\
  	; BUILD=$x grep "^#### .*|" Makefile | sed -e 's/.*|//' $x \\\\\\
! 	; use=$x basename "\$\@" -all $x; n=$x echo \$\$BUILD | wc -w $x \\\\\\
  	; echo "MAKE \$\$HOST : \$\$n * \$\@ (all local builds)" \\\\\\
  	; test ".\$\$BUILD" = "." && BUILD="." \\\\\\
  	; for i in \$\$BUILD ; do test ".\$\$i" = "." && continue \\\\\\
--- 16377,16383 ----
  	@ HOST="\$(HOST)\" \\\\\\
  	; test ".\$\$HOST" = "." && HOST=$x sh $ax_enable_builddir_auxdir/config.guess $x \\\\\\
  	; BUILD=$x grep "^#### .*|" Makefile | sed -e 's/.*|//' $x \\\\\\
! 	; use=$x basename "\$\@" -all $x; n=$x echo \$\$BUILD | wc -w $x  | tr -d ' ' \\\\\\
  	; echo "MAKE \$\$HOST : \$\$n * \$\@ (all local builds)" \\\\\\
  	; test ".\$\$BUILD" = "." && BUILD="." \\\\\\
  	; for i in \$\$BUILD ; do test ".\$\$i" = "." && continue \\\\\\
