*** configure.orig	Thu Nov 10 06:00:06 2011
--- configure	Thu Nov 10 06:03:15 2011
***************
*** 16338,16344 ****
  	@ HOST="\$(HOST)\" \\\\\\
  	; test ".\$\$HOST" = "." && HOST=$x sh $ax_enable_builddir_auxdir/config.guess $x \\\\\\
  	; BUILD=$x grep "^#### \$\$HOST " Makefile | sed -e 's/.*|//' $x \\\\\\
! 	; use=$x basename "\$\@" -all $x; n=$x echo \$\$BUILD | wc -w $x \\\\\\
  	; echo "MAKE \$\$HOST : \$\$n * \$\@"; if test "\$\$n" = "0" ; then : \\\\\\
  	; BUILD=$x grep "^####.*|" Makefile |tail -1| sed -e 's/.*|//' $x ; fi \\\\\\
  	; test ".\$\$BUILD" = "." && BUILD="." \\\\\\
--- 16338,16344 ----
  	@ HOST="\$(HOST)\" \\\\\\
  	; test ".\$\$HOST" = "." && HOST=$x sh $ax_enable_builddir_auxdir/config.guess $x \\\\\\
  	; BUILD=$x grep "^#### \$\$HOST " Makefile | sed -e 's/.*|//' $x \\\\\\
! 	; use=$x basename "\$\@" -all $x; n=$x echo \$\$BUILD | wc -w | tr -d ' ' $x \\\\\\
  	; echo "MAKE \$\$HOST : \$\$n * \$\@"; if test "\$\$n" = "0" ; then : \\\\\\
  	; BUILD=$x grep "^####.*|" Makefile |tail -1| sed -e 's/.*|//' $x ; fi \\\\\\
  	; test ".\$\$BUILD" = "." && BUILD="." \\\\\\
***************
*** 16371,16377 ****
  	@ HOST="\$(HOST)\" \\\\\\
  	; test ".\$\$HOST" = "." && HOST=$x sh $ax_enable_builddir_auxdir/config.guess $x \\\\\\
  	; BUILD=$x grep "^#### .*|" Makefile | sed -e 's/.*|//' $x \\\\\\
! 	; use=$x basename "\$\@" -all $x; n=$x echo \$\$BUILD | wc -w $x \\\\\\
  	; echo "MAKE \$\$HOST : \$\$n * \$\@ (all local builds)" \\\\\\
  	; test ".\$\$BUILD" = "." && BUILD="." \\\\\\
  	; for i in \$\$BUILD ; do test ".\$\$i" = "." && continue \\\\\\
--- 16371,16377 ----
  	@ HOST="\$(HOST)\" \\\\\\
  	; test ".\$\$HOST" = "." && HOST=$x sh $ax_enable_builddir_auxdir/config.guess $x \\\\\\
  	; BUILD=$x grep "^#### .*|" Makefile | sed -e 's/.*|//' $x \\\\\\
! 	; use=$x basename "\$\@" -all $x; n=$x echo \$\$BUILD | wc -w $x  | tr -d ' ' \\\\\\
  	; echo "MAKE \$\$HOST : \$\$n * \$\@ (all local builds)" \\\\\\
  	; test ".\$\$BUILD" = "." && BUILD="." \\\\\\
  	; for i in \$\$BUILD ; do test ".\$\$i" = "." && continue \\\\\\
