*** pango/Makefile.in.orig	Wed May 30 06:49:09 2012
--- pango/Makefile.in	Wed May 30 06:55:26 2012
***************
*** 56,62 ****
  # ------------------- libpangoxft -------------------
  @HAVE_XFT_TRUE@am__append_12 = $(pangoxft_headers)
  @HAVE_XFT_TRUE@am__append_13 = libpangoxft-1.0.la
! @HAVE_XFT_TRUE@am__append_14 = PangoXft-1.0.gir
  @PLATFORM_WIN32_TRUE@am__append_15 = -export-symbols $(srcdir)/pangoxft.def -Wl,pangoxft-win32-res.o
  @PLATFORM_WIN32_TRUE@am__append_16 = pangoxft-win32-res.o pangoxft.def
  
--- 56,62 ----
  # ------------------- libpangoxft -------------------
  @HAVE_XFT_TRUE@am__append_12 = $(pangoxft_headers)
  @HAVE_XFT_TRUE@am__append_13 = libpangoxft-1.0.la
! @HAVE_XFT_TRUE@am__append_14 =
  @PLATFORM_WIN32_TRUE@am__append_15 = -export-symbols $(srcdir)/pangoxft.def -Wl,pangoxft-win32-res.o
  @PLATFORM_WIN32_TRUE@am__append_16 = pangoxft-win32-res.o pangoxft.def
  
***************
*** 732,738 ****
  
  PangoFT2_1_0_gir_PACKAGES = freetype2
  PangoFT2_1_0_gir_EXPORT_PACKAGES = pangoft2
! PangoFT2_1_0_gir_INCLUDES = GObject-2.0 cairo-1.0 freetype2-2.0 fontconfig-2.0
  PangoFT2_1_0_gir_LIBS = libpangoft2-1.0.la
  PangoFT2_1_0_gir_CFLAGS = -I$(top_srcdir) -I$(top_builddir)
  PangoFT2_1_0_gir_SCANNERFLAGS = --include-uninstalled=$(builddir)/Pango-1.0.gir --c-include "pango/pangoft2.h"
--- 732,738 ----
  
  PangoFT2_1_0_gir_PACKAGES = freetype2
  PangoFT2_1_0_gir_EXPORT_PACKAGES = pangoft2
! PangoFT2_1_0_gir_INCLUDES = GObject-2.0 cairo-1.0 freetype2-2.0
  PangoFT2_1_0_gir_LIBS = libpangoft2-1.0.la
  PangoFT2_1_0_gir_CFLAGS = -I$(top_srcdir) -I$(top_builddir)
  PangoFT2_1_0_gir_SCANNERFLAGS = --include-uninstalled=$(builddir)/Pango-1.0.gir --c-include "pango/pangoft2.h"
